src/components/Header.js

```diff
@@ -7,7 +10,8 @@ import {
 import { MainMenu } from '../MainMenu';
 import FixedMenu from '../FixedMenu';
 
-const Header = ({fix}) => {
+const Header = ({fix, location}) => {
+  if (location.pathname === '/') return (<div></div>)
```

Comment:
This is hax. I wrote it, so the shame is mine, but there ought to be a fast follow on issue here to find a better way to not display the header on the home page.